src/olympia/stats/tests/test_utils.py

```diff
@@ -435,3 +477,85 @@ def test_returned_values(self, bigquery_client_mock):
             'guid': {'avg_this_week': 123, 'avg_three_weeks_before': 456},
             'guid2': {'avg_this_week': 45, 'avg_three_weeks_before': 40},
         }
+
+
+@override_settings(BIGQUERY_PROJECT='project', BIGQUERY_AMO_DATASET='dataset')
+class TestGetDownloadSeries(BigQueryTestMixin, TestCase):
+    def setUp(self):
+        super().setUp()
+
+        self.addon = addon_factory()
+
+    @mock.patch('google.cloud.bigquery.Client')
+    def test_create_client(self, bigquery_client_mock):
+        client = self.create_mock_client()
+        bigquery_client_mock.from_service_account_json.return_value = client
+
+        credentials = 'path/to/credentials.json'
+        with override_settings(GOOGLE_APPLICATION_CREDENTIALS=credentials):
+            get_download_series(
+                addon=self.addon,
+                start_date=date(2020, 5, 27),
+                end_date=date(2020, 5, 28),
+            )
+
+        bigquery_client_mock.from_service_account_json.assert_called_once_with(
+            credentials
+        )
+
+    @mock.patch('olympia.stats.utils.statsd.timer')
+    @mock.patch('google.cloud.bigquery.Client')
+    def test_create_query(self, bigquery_client_mock, timer_mock):
+        client = self.create_mock_client()
+        bigquery_client_mock.from_service_account_json.return_value = client
+        start_date = date(2020, 5, 27)
+        end_date = date(2020, 5, 28)
+        expected_query = f"""
+SELECT submission_date, total_downloads
+FROM `project.dataset.{AMO_STATS_INSTALLS_VIEW}`
+WHERE addon_id = @addon_id
+AND submission_date BETWEEN @submission_date_start AND @submission_date_end
+ORDER BY submission_date DESC
+LIMIT 365"""
+
+        get_download_series(
+            addon=self.addon, start_date=start_date, end_date=end_date
+        )
+
+        client.query.assert_called_once_with(
+            expected_query, job_config=mock.ANY
```

Comment:
Shouldn't we also verify the `query_parameters` part of `job_config`? That seems just as important as verifying that the `query` was correct.

Reply:
Yep, good idea. I didn't have a good way to assert `job_config` back then but now I have a helper so I'll update this test case and the similar one for `get_updates_series`